### ISSUE TITLE
Windows console does not show fira code 6.2 on the list of fonts that can be used.

### ISSUE THREAD

**Issue by createdbyjurand** [tonsky/FiraCode#1409]
Previous versions of Fira Code worked. 
Something changed that Windows is not accepting it anymore?
Other programs see it and use it.
Only Windows Console and PowerShell Terminal old and new do not display it on the list.

![cmd](https://user-images.githubusercontent.com/32194442/167586240-7542606f-d3a4-4901-9017-6e14c2ab3c39.png)

**Comment by tonsky** [comment]
I have no idea

**Comment by anilkumar150886** [comment]
Yeah.. its the same for me too.. I have the latest Fira Code that is installed and it disappeared from Font List in Cmd.. 
In 5.01 version, it was working.. 

![image](https://github.com/tonsky/FiraCode/assets/16396710/9cb07f86-47b4-4d8c-924f-90da0298802b)

**Comment by grenelt** [comment]
Still is now showing up...

![grafik](https://github.com/user-attachments/assets/f6213aaf-36c8-4533-9803-3f53a9dff114)
